tests/integration/integration_test_endpoints/server/raw.py

```diff
@@ -191,3 +194,34 @@ def test_celery_task_revoke_in_queue(self, params):
             blocker.revoke()
 
         return result.job
+
+    @access.token
+    @filtermodel(model='job', plugin='jobs')
+    @describeRoute(
+        Description('Test chained celery tasks')
+    )
+    def test_celery_task_chained(self, params):
+        jobModel = ModelImporter.model('job', 'jobs')
+        user = self.getCurrentUser()
+        # F(F(F(6))) --> F(F(8)) --> F(21) --> 10946
+        result = (fibonacci.s(6) | fibonacci.s() | fibonacci.s()).delay()
+        result.wait(timeout=10)
+        job_1 = result.job
+        job_2 = jobModel.load(job_1['parentId'], user=user)
+        job_3 = jobModel.load(job_2['parentId'], user=user)
+
+        return [job_1, job_2, job_3]
+
+    @access.token
+    @filtermodel(model='job', plugin='jobs')
+    @describeRoute(
+        Description('Test chained celery tasks with a bad token')
+    )
+    def test_celery_task_chained_bad_token_fails(self, params):
+        # By default the first task (fib(5)) should try to set it's token
+        # to the second task. If user passes a token, it should not overwrite
+        # the given token though.
+
+        result = (fibonacci.s(5) | fibonacci.si(5, girder_client_token='')).delay()
```

Comment:
Can we test this against a task that hits an endpoint that actually requires the authorization of a specific girder_client_token?